pkg/computeclasses/computeclasses.go

```diff
@@ -134,18 +129,6 @@ func CalculateCPU(cc internaladminv1.ProjectComputeClassInstance, memDefault *in
 	return *resource.NewMilliQuantity(int64(math.Ceil(value)), resource.DecimalSI), nil
 }
 
-func ValidateProjectComputeClass(cc internaladminv1.ProjectComputeClassInstance, memory resource.Quantity, memDefault *int64) error {
-	return Validate(apiv1.ComputeClass{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: cc.Name,
-		},
-		Memory:           cc.Memory,
-		Description:      cc.Description,
-		Default:          cc.Default,
-		SupportedRegions: cc.SupportedRegions,
-	}, memory, memDefault)
-}
-
```

Comment:
This function was only called in the above line that was removed, so I'm taking it out